tests/integ/test_dataframe.py

```diff
@@ -515,6 +522,10 @@ def process(self, n: int):
     Utils.check_answer(df.select(df.a, table_func(df.b, lit(" "))), expected_result)
 
 
+@pytest.mark.skipif(
+    "config.getvalue('local_testing_mode')",
+    reason="TODO: Session.generator is not supported in Local Testing",
```

Comment:
Does this one have a ticket number? Same with test_generator_table_function_negative.